.storybook/stories/button/button-group.stories.ts

```diff
@@ -48,6 +48,7 @@ const defaultParameters: Parameters = {
     createArray: { control: { disable: true }, table: { disable: true } },
     buttonCount: { control: { type: 'number', min: 1, max: 100 } },
     inMenuButtonCount: { control: { type: 'number', min: 1, max: 100 } },
+    disabledButtonsPosition: { description: 'Enter comma separated numbers (1,2..)', control: { type: 'text' } },
```

Comment:
```suggestion
    disabledButtonsPosition: { description: 'Enter JSON array (e.g. `[2,3]`)', control: { type: 'array' } },
```

We can use a native array control so that we don't have to call `split` in the template.

Other changes:
1. Set the default value to `[]` instead of `''`.
2. Use in `[disabled]="disabledButtonsPosition.includes(i+1)"` in the template.